apps/mobile/src/libs/components/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,74 @@
+import CheckBox from "@react-native-community/checkbox";
+import React from "react";
+
+import { LinearGradient, Text, View } from "~/libs/components/components";
+import { BaseColor, GradientColor, GradientPoint } from "~/libs/enums/enums";
+import { globalStyles } from "~/libs/styles/global-styles/global-styles";
+
+import { styles } from "./styles";
+
+type Properties = {
+	isChecked: boolean;
+	label?: string;
+	name?: string;
+	onValueChange: (newValue: boolean) => void;
+	restProperties?: Record<string, unknown>;
```

Comment:
This is not a correct way to execute all other LibraryCheckbox props. Import `{ type CheckBoxProps } from "@react-native-community/checkbox"` and use it
```
type Properties = CheckBoxProps & {
    isChecked: boolean;
  	label: string;
}
```